plugins/circleci/README.md

```diff
@@ -63,6 +63,19 @@ proxy:
 5. Get and provide `CIRCLECI_AUTH_TOKEN` as env variable (https://circleci.com/docs/api/#add-an-api-token)
 6. Add `circleci.com/project-slug` annotation to your component-info.yaml file in format <git-provider>/<owner>/<project> (https://backstage.io/docs/architecture-decisions/adrs-adr002#format)
 
+```
```

Comment:
oh hey nitpick - it's nice to start this with ` ```yaml ` so you get syntax highlighting

Reply:
Fixed. It's two commits now but I assume you can squash them when you merge?

Reply:
Yupyup, no problem